src/main/js/Navigation.js

```diff
@@ -40,12 +40,18 @@ class Navigation extends Component {
 
     render() {
         const channelGlobals = this.createNavItemForDescriptors(DescriptorUtilities.DESCRIPTOR_TYPE.CHANNEL, DescriptorUtilities.CONTEXT_TYPE.GLOBAL, '/alert/channels/', 'Channels');
-        const providers = this.createNavItemForDescriptors(DescriptorUtilities.DESCRIPTOR_TYPE.PROVIDER, DescriptorUtilities.CONTEXT_TYPE.GLOBAL, '/alert/providers/', 'Providers');
         const components = this.createNavItemForDescriptors(DescriptorUtilities.DESCRIPTOR_TYPE.COMPONENT, DescriptorUtilities.CONTEXT_TYPE.GLOBAL, '/alert/components/');
 
         const nav = (
             <Fragment>
-                {providers}
+                <li className="navHeader">
+                    Providers
+                </li>
+                <li>
+                    <NavLink to="/alert/providers/blackduck" activeClassName="activeNav">
+                        Black Duck
+                    </NavLink>
```

Comment:
Anything we can do to make this more dynamic? We should have a display name in the provider metadata no?

Reply:
I should be able to iterate through the providers to make it more dynamic.